src/main/java/lotto/domain/WinningRanks.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class WinningRanks {
+    private static final int WINNING_AMOUNT = 1;
+    // todo: 이게 무슨 의미인 지 조금 더 명확하면 좋겠다.
+    private static final int DEFAULT_WINNING_TICKET_SIZE = 0;
+    private Map<Rank, Integer> winningRanks;
+
+    WinningRanks(Map<Rank, Integer> winningRanks) {
+        this.winningRanks = winningRanks;
+    }
+
+    public Map<Rank, Integer> getWinningRanks() {
+        return winningRanks;
+    }
+
+    /* winningTicketSize은 각 Rank별 당첨된 티켓 수를 의미한다. */
+    Map<Rank, Integer> addWinningRanks(Rank rank) {
+        Integer winningTicketSize = winningRanks.get(rank);
+        if (winningTicketSize == null) {
```

Comment:
```suggestion
        if (Objects.isNull(winningTicketSize)) {
```

저는 null check를 할때 Objects클래스에서 제공해주는 메소드를 사용하는 편이에요.
https://multifrontgarden.tistory.com/205

Reply:
그렇군요 감사합니다!